domains/certificates/Query.API/API.AppTests/Infrastructure/QueryApiWebApplicationFactory.cs

```diff
@@ -24,24 +25,44 @@ public class QueryApiWebApplicationFactory : WebApplicationFactory<Program>
     public string MartenConnectionString { get; set; } = "";
     public string DataSyncUrl { get; set; } = "";
 
+    public RabbitMqOptions RabbitMqSetup { get; set; }
+
+
     protected override void ConfigureWebHost(IWebHostBuilder builder)
     {
         builder.UseSetting("ConnectionStrings:Marten", MartenConnectionString);
         builder.UseSetting("Datasync:Url", DataSyncUrl);
 
+        if (IsRabbitMqSet())
```

Comment:
I think that as a default the ASPNETCORE_ENVIRONMENT will be set to development. If that is right, the application **may** (I am not sure) use the values listed in appsettings.development.json in QueryAPI-project. 
If not overwritten (e.g. when IsRabbitSet() is false), then will it have null/empty values or use the ones from appsettings.development.json? I am not sure. But if it uses appsettings.development.json values, then someone running the test locally will have strange test results if there is a rabbitmq running.

Doest it make sense?

Reply:
Not sure if it make sense, but no problems here when running a rabbitmq while running the tests